cli/internal/terraform/variables.go

```diff
@@ -74,8 +74,8 @@ type AWSNodeGroup struct {
 	// StateDiskSizeGB is the size of the state disk to allocate to each node, in GB.
 	StateDiskSizeGB int `hcl:"disk_size" cty:"disk_size"`
 	// InitialCount is the initial number of nodes to create in the node group.
-	// During upgrades this value is not set.
-	InitialCount *int `hcl:"initial_count" cty:"initial_count"`
+	// During upgrades this value ignored.
+	InitialCount int `hcl:"initial_count" cty:"initial_count"`
```

Comment:
I found it rather confusing to use since the same variable is passed around over multiple layers and is only considered a pointer type here.
The name makes it very obvious that this is only used for the *initial* count so I believe this is easier and safer (cannot be nil, no chance to deref a nil ptr).